tests/fixtures/_utils.js

```diff
@@ -1,217 +1,336 @@
+
+      // if it's an array, we're in a repeatable - recurseit!
+      if (Array.isArray(item)) {
+        referencer(t, item);
+
+        return;
+      }
 
-    const id = uuid.v4();
-    const date = moment().format('YYYY-MM-DD');
+      // non-repeatable attribute with multiple inputs
+      if (typeof item === 'object' && typeof item[Object.keys(item)[0]] === 'object') {
+        // if it has attributes, we're not at depth so....uuuuuuhhhhhhhaaaAAAAHHHH recursit!
+        if (item[Object.keys(item)[0]].hasOwnProperty('attributes')) {
+          Object.keys(item).forEach(itm => {
+            referencer(t, item[itm].attributes);
+          });
 
-    const sunrise = moment().format('hh:mm');
-    const sunset = moment().format('hh:mm');
+          return;
+        }
 
-    const type = types[Math.round(Math.random() * 4)];
+        // no attributes - it should have a meta then
+        Object.keys(item).forEach(itm => {
+          t.true(item[itm].hasOwnProperty('meta'), 'attribute in array contains meta');
 
-    eachOfType[type] += 1;
+          return;
+        });
+      }
 
-    if (content.length > 0) {
-      // make the last entry the referencee
-      referenced = content[content.length - 1].id;
+      // non-repeatable; single input
+      else {
+        t.true(item.hasOwnProperty('meta'), 'attribute in array contains meta');
+
+        return;
+      }
     }
+  });
+};
 
-    const values = {};
-    values[`${slugify(type)}-name`] = {
-      text: {
-        value: ipsum({
-          count: 1,
-          units: 'words',
-          format: 'plain',
-        }),
-      },
-    };
-    values[`${slugify(type)}-textblock`] = {
-      textblock: {
-        value: ipsum({
-          count: 2,
-          units: 'paragraphs',
-          format: 'plain',
-          sentenceUpperBound: 5,
-          paragraphUpperBound: 3,
-        }),
-      },
-    };
-    values[`${slugify(type)}-referencer`] = {
-      referencer: {
-        value: referenced,
-      },
-    };
-
-    const item = {
-      id,
-      language,
-      'revision': Math.round(Math.random() * 40),
-      'sunrise': utils.time.iso(date, sunrise, 'America/New_York'),
-      'sunrise-timezone': 'America/New_York',
-      'sunset': Date.now() % 6 === 0 ? utils.time.iso(date, sunset, 'America/New_York') : null,
-      'sunset-timezone': 'America/New_York',
-      'identifier': name,
-      'slug': slugify(name),
-      type,
-      'type-slug': slugify(type),
-      'value': values,
-    };
-
-    content[i] = item;
-
-    const live = {
-      id,
-      language,
-      'sunrise': utils.time.iso(date, sunrise, 'America/New_York'),
-      'sunset': Date.now() % 6 === 0 ? utils.time.iso(date, sunset, 'America/New_York') : null,
-      'attributes': values,
-      'key': name,
-      'key-slug': slugify(name),
-      type,
-      'type-slug': slugify(type),
-      'revision': Math.round(Math.random() * 40),
-    };
-
-    lives[i] = live;
+const attribute = (t, attr) => {
```

Comment:
This needs a better name. Its name suggests checking attributes, where its checking formatted.